can-make-rest.js

```diff
@@ -35,8 +35,8 @@ function getItemAndListUrls (url, idProp) {
 
 module.exports = function(url, idProp){
 	var data= {};
-	each( getItemAndListUrls(url, idProp), function(url, type){
-		each(methodMapping[type], function(interfaceMethod, method){
+	canReflect.each( getItemAndListUrls(url, idProp), function(url, type){
```

Comment:
generally speaking, I use `canReflect.eachKey` instead of `canReflect.each` to avoid having to detect how to loop through the value.  Keeps things slightly faster.

Reply:
Thanks for the tip!

Reply:
Done!